### PR TITLE
Task-37850: Integrate the DLP keywords file via UI

### PR DESCRIPTION
**Pull request by AzmiTouil**

### CODE REVIEW

**Review by rdenarie — DISMISSED** [review]

**Review by rdenarie — DISMISSED** [review]

File: commons-dlp/src/main/java/org/exoplatform/commons/dlp/service/impl/DlpPositiveItemServiceImpl.java [hunk 1]
```diff
@@ -83,6 +94,12 @@ public Long getDlpPositiveItemsCount() {
     return dlpPositiveItemDAO.count();
   }
 
+  @Override
+  public String getDlpKeywords() {
+    SettingValue<?> settingValue = settingService.get(Context.GLOBAL, Scope.GLOBAL.id("DlpKeywords"), EXO_DLP_KEYWORDS);
+    return settingValue != null ? settingValue.getValue().toString() : System.getProperty(DLP_KEYWORDS);
+  }
+
```
Comment (rdenarie) [inline commons-dlp/src/main/java/org/exoplatform/commons/dlp/service/impl/DlpPositiveItemServiceImpl.java]: Here we have duplicated code. getDlpKeyword is present in DlpServiceConnector  and here.
I think it should be moved to DlpService which is the main service for DLP
DlpPositiveItemService is for managing DlpPositivieItems

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: commons-dlp/src/main/java/org/exoplatform/commons/dlp/service/impl/DlpPositiveItemServiceImpl.java [hunk 2]
```diff
@@ -1,6 +1,7 @@
 package org.exoplatform.commons.dlp.service.impl;
```
Comment (rdenarie) [inline commons-dlp/src/main/java/org/exoplatform/commons/dlp/service/impl/DlpPositiveItemServiceImpl.java]: Modifications in this file should be removed I think

File: commons-dlp/src/test/java/org/exoplatform/commons/dlp/service/DlpPositiveItemServiceTest.java [hunk 3]
```diff
@@ -1,5 +1,6 @@
 package org.exoplatform.commons.dlp.service;
```
Comment (rdenarie) [inline commons-dlp/src/test/java/org/exoplatform/commons/dlp/service/DlpPositiveItemServiceTest.java]: Modifications in this file should be removed

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: commons-dlp/src/main/java/org/exoplatform/commons/dlp/service/impl/DlpPositiveItemServiceImpl.java (hunk 2 above)
Comment (AzmiTouil) [reply commons-dlp/src/main/java/org/exoplatform/commons/dlp/service/impl/DlpPositiveItemServiceImpl.java]: Done!

**Review by ahmadmido — APPROVED** [review]
LGTM

**Review by rdenarie — APPROVED** [review]

**Pull request merged**